src/common/base/test/CMakeLists.txt

```diff
@@ -111,7 +111,11 @@ nebula_link_libraries(
 )
 nebula_add_test(nebulakey_utils_test)
 
-add_executable(range_vs_transform_bm RangeVsTransformBenchmark.cpp)
+add_executable(
+    range_vs_transform_bm
+    RangeVsTransformBenchmark.cpp
+    $<TARGET_OBJECTS:base_obj>
```

Comment:
This binary is being linked before the folly library is ready, right?

Reply:
Yes, this binary is depend on folly. But folly library wasn't compiled at the time.
```
[  0%] Building CXX object src/common/base/test/CMakeFiles/range_vs_transform_bm.dir/RangeVsTransformBenchmark.cpp.o
/media/psf/Home/work/source_code/fix_parallel_compilation_error/src/common/base/test/RangeVsTransformBenchmark.cpp:6:10: fatal error: folly/init/Init.h: No such file or directory
 #include <folly/init/Init.h>
          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
```